docker/orchestrator/build.sh

```diff
@@ -19,14 +19,14 @@ set -e
 tmpdir=`mktemp -d`
 
 script="go get vitess.io/vitess/go/cmd/vtctlclient && \
-  git clone https://github.com/github/orchestrator.git src/github.com/github/orchestrator && \
-  go install github.com/github/orchestrator/go/cmd/orchestrator"
+  git clone https://github.com/openark/orchestrator.git src/github.com/openark/orchestrator && \
+  go install github.com/openark/orchestrator/go/cmd/orchestrator"
 
 echo "Building orchestrator..."
-docker run -ti --name=vt_orc_build golang:1.7 bash -c "$script"
+docker run -ti --name=vt_orc_build golang:1.14.4-stretch bash -c "$script"
```

Comment:
current vitess minimum version is 1.13. Do we want to keep that same here?

Reply:
I'm not sure if this docker image is even being built or used anywhere

Reply:
I believe it is not being used officially. However, users who hav ecloned this repo locally may try and use it (it works).

Reply:
> current vitess minimum version is 1.13.

Latest `orchestrator` has a `go1.14` minimum requirement. I guess for `3.1.0` we can use `1.13` -- but now I notice this image uses an old version of `orchestrator`... I'll update that, too.